llvm/include/llvm/IR/Instruction.h

```diff
@@ -44,15 +44,19 @@ template <> struct ilist_alloc_traits<Instruction> {
 iterator_range<simple_ilist<DbgRecord>::iterator>
 getDbgRecordRange(DbgMarker *);
 
+/// Class used to generate an insert position (ultimately always a
+/// BasicBlock::iterator, which it will implicitly convert to) from either:
+/// - An Instruction, inserting immediately prior.
```

Comment:
Indicate that this is deprecated?

Reply:
It isn't deprecated yet, but I can put a comment indicating the future intent.